src/includes/performance/beforeNavigate-example/javascript.mdx

```diff
@@ -1,9 +1,8 @@
-```javascript
-import * as Sentry from "@sentry/browser";
-import { Integrations } from "@sentry/tracing";
+One common use case is parameterizing transaction names. For both `pageload` and `navigation` transactions, the `BrowserTracing` integration uses the browser's `window.location` value to generate a transaction name. Using `beforeNavigate` you can modify the transaction name to make it more generic, so that, for example, transactions named `GET /users/12312012` and `GET /users/11212012` can both be renamed `GET /users/:userid`, so that they'll group together.
```

Comment:
This paragraph should also be included for platform-specific content (e.g. also Next.js).

Reply:
I specifically didn't include it for nextjs because we automatically parameterize transaction names, for any transaction `beforeNavigate` would handle, since manual transactions don't pass through it, and we handle both pageload and navigation transaction-name parameterization. 

For all other frameworks, them not having a file of their own means they'll use this one.